priv/leveled.schema

```diff
@@ -143,7 +149,7 @@
 %% then it is a candidate (e.g. in default case if 50% of space would be
```

Comment:
```suggestion
%% then it is a candidate (e.g. in default case if 50% of space would be
```
```suggestion
%% then it is a candidate (e.g. in default case if 30% of space would be
```

Reply:
Amended - note that the percentage is the inverse of the space released